rootfs/rootfs/etc/rc.d/automount-shares

```diff
@@ -6,7 +6,7 @@ set -e
 # VirtualBox Guest Additions
 # - this will bail quickly and gracefully if we're not in VBox
 if modprobe vboxguest &> /dev/null && modprobe vboxsf &> /dev/null; then
-	mountOptions='defaults'
+	mountOptions='iocharset=utf8'
```

Comment:
According to the [`mount.vboxsf` man page](http://manpages.ubuntu.com/manpages/trusty/man8/mount.vboxsf.8.html), `utf8` is the default `iocharset`, so I'm a little confused why this would be necessary.